flink-connectors/flink-connector-kafka-0.10/src/test/java/org/apache/flink/streaming/connectors/kafka/Kafka010ITCase.java

```diff
@@ -168,8 +168,8 @@ public void testAutoOffsetRetrievalAndCommitToKafka() throws Exception {
 
 	/**
 	 * Kafka 0.10 specific test, ensuring Timestamps are properly written to and read from Kafka.
+	 * This test is disabled because of: https://issues.apache.org/jira/browse/FLINK-9217
 	 */
-	@Test(timeout = 60000)
```

Comment:
Let's ignore the test with `@Ignore` and a proper ignore message. This should make it easier to find it by looking for `@Ignore` annotated tests.

Reply:
Ops, I was looking for a away to disable the test via `@Test(enabled = false)` or sth like this. Missed that. Fixed